packages/quantic/force-app/main/default/lwc/quanticTab/quanticTab.css

```diff
@@ -0,0 +1,5 @@
+.slds-tabs_button{
```

Comment:
Are you overriding a slds default class?
You should not override a slds class,
and if it's not a slds class, you should be using a different nomenclature for your classes
Example: `.tab_button`